src/VsixV3/Microsoft.Net.Sdk.swr

```diff
@@ -0,0 +1,80 @@
+use vs
+
+package name=Microsoft.Net.Sdk
+        version=$(Version)
+
+vs.localizedResources
+  vs.localizedResource language=en-us
+                       title="Microsoft Net Sdk"
+                       description="SDK for .NET projects"
+
+folder "InstallDir:MSBuild\Sdks\Microsoft.Net.Sdk\Sdk"
```

Comment:
All the folders with Microsoft.Net.Sdk in them need to be changed to Microsoft.NET.Sdk